app/modules/projects/models.py

```diff
@@ -6,10 +6,22 @@
 
 from sqlalchemy_utils import Timestamp
 from app.extensions import db, HoustonModel
-
+from app.modules.encounters import models as encounters_models  # NOQA
 import uuid
 
 
+# todo should this be a class or an association table
```

Comment:
i am curious to hear from @bluemellophone about this myself.  i _totally_ get what your intention is here -- using this as a join table for many-to-many ... but just have no clue/experience as to the right way to do this.  obviously i am not the best reviewer for _this_ part then. haha  but will not let that stop me?

Reply:
Yeah, I'm curious about this also. Do you need to explicitly define a join table object for all many to many relationships or are there some other benefits to this? I can see value in adding extra info like a timestamp to the association object.

Reply:
@colinwkingen I understand that you do need an association table for all many:many relationships. What I'm not certain about is if this needs to be a class or just a table. @bluemellophone created a class for the user to org mapping to allow  extra checks in future. For user to project, that also makes sense. For Project to Encounter, I'm less sure, hence the question.